pkg/crc/network/utils.go

```diff
@@ -92,3 +94,17 @@ func UriStringForDisplay(uri string) (string, error) {
 	}
 	return uri, nil
 }
+
+func CheckCRCLocalDNSReachableFromHost(crcbundleMetadata *bundle.CrcBundleInfo) error {
```

Comment:
Wondering if we need the dependency on the `bundle`package, or if this should be
` func CheckCRCLocalDNSReachableFromHost(domainName, appsDomainName string) error` ?